packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -236,10 +248,19 @@ export default {
 
     .caret-wrapper {
         float: left;
-        display: inline-block;
         width: 32px;
     }
 
+    .minimal-caret-wrapper {
+        display: inline-block;
+        font-size: 13px;
+        margin-right: 2px;
```

Comment:
Might want to slightly increase the margin between the minimized switch and the card title on mobile? Right now it looks a little too cluttered.

![image](https://user-images.githubusercontent.com/47470981/150746387-0dbdf115-c1bc-4c7f-b030-b8053f33a0de.png)